python-api/products_ranker_api/tests/test_URL_crawling.py

```
@@ -0,0 +1,47 @@
+    try:
+        SearchRecipe(
+            uuid="abc-123",
+            name="invalid",
+            keywords=["usb"],
+            min_price=50.0,
+            max_price=10.0
+        )
+    except ValidationError:
+        assert True
+    else:
+        assert False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)